sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -14,7 +14,7 @@ import {
   TokenCredential,
   SharedKeyCredential
 } from "@azure/core-amqp";
-import { ServiceBusClientOptions } from "./serviceBusClient";
+import { ServiceBusClientOptions } from "./old/serviceBusClient";
```

Comment:
Seeing "old" as a folder path makes me thing that the contents of this folder are deprecated. What's the expected outcome in this case? are we thinking on removing this folder eventually?

Reply:
If the "old" folder is where the generated code is, perhaps we can make an issue to rename it to "generated" or something like that? Just a thought.

Reply:
"old" has the older API, which is being used for the tests.
We'll soon be deleting the "old" folder(in a future PR) and update all the tests to leverage track 2 code.

Reply:
Yeah, just another temporary measure. The old code had the best names so I just wanted it be clear which one I was using as I was moving things around. :)